src/main/perl/Lock.pm

```diff
@@ -9,21 +9,22 @@ use strict;
 use CAF::Object;
 use CAF::Reporter;
 
-use LC::Exception qw (SUCCESS throw_error);
+use LC::Exception qw(SUCCESS throw_error);
 use FileHandle;
-#use Proc::ProcessTable;
+use Fcntl qw(:flock);
 
 use Exporter;
 use vars qw(@ISA @EXPORT @EXPORT_OK);
 
 @ISA = qw(CAF::Reporter CAF::Object Exporter);
 
-@EXPORT_OK = qw(FORCE_ALWAYS FORCE_IF_STALE);
+@EXPORT_OK = qw(FORCE_NONE FORCE_ALWAYS FORCE_IF_STALE);
 
 
-use constant FORCE_NONE => 0;
-use constant FORCE_ALWAYS => 1;
-use constant FORCE_IF_STALE => 2;
+use constant FORCE_NONE     => 0;
+use constant FORCE_ALWAYS   => 1;
+use constant FORCE_IF_STALE => 2;  # for backwards compatibility only
```

Comment:
if this value is used somewhere, at least report it with `verbose` or `info`, and mention that it has no effect and that developers should be contacted.